iina/MainWindowController.swift

```diff
@@ -2265,6 +2262,16 @@ class MainWindowController: PlayerWindowController {
     }
   }
 
+  private func refreshSeekTimeAndThumnail(from event: NSEvent) {
+    let isCoveredByOSD = !osdVisualEffectView.isHidden && isMouseEvent(event, inAnyOf: [osdVisualEffectView])
```

Comment:
I also found that if the sidebar covers the thumbnail, it will also trigger the thumbnail, and I've confirmed that something like this
`let isCoveredBySidebar = !sideBarView.isHidden && isMouseEvent(event, inAnyOf: [sideBarView])`
and checking both `isCoveredByOSD` and `isCoveredBySidebar` is able to fix this problem